src/table.js

```diff
@@ -479,6 +462,27 @@ function likeOperand(operand) {
   return {...operand, value: `%${operand.value}%`};
 }
 
+// Check if a value is a valid form of the column type
+export function isValid(value, colType) {
+  switch (colType) {
+    case "string":
+    case "bigint":
+    case "boolean":
+      return typeof value === colType;
+    case "number":
+      return typeof value === colType && !Number.isNaN(value);
```

Comment:
You can simplify here, too, since at this point we already know that `value` is a number and hence will not require coerce.

```suggestion
      return typeof value === colType && !isNaN(value);
```

Reply:
Maybe I'm mistaken, but I thought that `isNaN` does the coercion, so here is used `Number.isNaN` to _avoid_ unnecessarily doing coercion.

Reply:
That’s correct. You can use either here and get the same result. The only difference is that one is shorter to type—that’s all I was saying. You’re welcome to leave it as-is.

Reply:
Ah, by "simplify", I thought you meant simplify the computation. 

- `Number.isNaN` seems "simpler" in terms of the operation (doesn't do the additionally complex step of coercion)
- `isNaN` is simpler in terms of number of characters 😄